### PR TITLE
Arpit | Artificial Neural Networks | Added README

### PR DESCRIPTION
**Pull request by aceking007**
#54 Added Documentation for Artificial Neural Networks in Deep Learning.

### CODE REVIEW

File: 9_Deep_Learning/Artificial_Neural_Networks/README.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+Validation
+- Use a small part of the dataset (20% in many cases) for validation.
+- Provide just the input values.
+- Carry out propagation to get the output values.
+- Calculate a score based on how well the output compares to actual values.
+- If score is good enough, Yay! Else, repeat training with more data points or change connection structure (network topology).
+
+
+
+Finally, [**here**](https://www.youtube.com/watch?v=ILsA4nyG7I0&ab_channel=BrandonRohrer) is a video by Brandon Rohrer that explains the functioning of a neural network in the simplest and most intuitive way possible.
```
Comment (NishkarshRaj) [inline 9_Deep_Learning/Artificial_Neural_Networks/README.md:66]: Links must be provided in References sub-topic. 
```suggestion
### References

[Deep Learning and Neural Networks](https://www.youtube.com/watch?v=ILsA4nyG7I0&ab_channel=BrandonRohrer) - a video by Brandon Rohrer that explains the functioning of a neural network in the simplest and most intuitive way possible.
```

**Review by NishkarshRaj — CHANGES_REQUESTED** [review]
Few suggestions:

* Mention links/referred material in a reference section in bottom.

* The images linked are on your repository called Resources and linked with absolute path.
Commit those images on this repository on `9_Deep_Learning/img` folder and then reference them via relative path like `img/filename.extension` rather than `github.com/username/filename`

**Comment by aceking007** [comment]
Cool. Working on it.

**Comment by aceking007** [comment]
> Few suggestions:
> 
> * Mention links/referred material in a reference section in bottom.
> * The images linked are on your repository called Resources and linked with absolute path.
>   Commit those images on this repository on `9_Deep_Learning/img` folder and then reference them via relative path like `img/filename.extension` rather than `github.com/username/filename`

Made all the changes. Added a few more resources to the references.  
Please review the changes.

**Review by NishkarshRaj — APPROVED** [review]
Great work @aceking007 😄 

Merged with master 👍

**Pull request merged**